### DISCUSSION TITLE
date a time parts of DateTime

### DISCUSSION THREAD

**Discussion by terenzif** [moment/luxon#1077] [Ideas]
I use momentjs since long time. I often find it to miss the equivalent of c# properties DateTime.Date and DateTime.TimeOfDay: they let me store a full DateTime in a single object (as it should be) and direct access to Date and Time components.

At the moment, to get time part of luxon DateTime object i have to (considering date a DateTime):
* luxon.Duration.fromMillis(date - date.startOf("day"))
* luxon.Duration.fromISOTime(date.toISOTime()) //did not work but it should!?

For accessing the date part is quite simpler:
* date.startOf("day")

In my proposal this should be:
* date.datePart
* date.timePart

I repeat, i ask myself all the time why it has not been implemented.

Greetings

**Comment by icambron** [comment]
I don't understand what types you're expecting datePart and timePart to return. In one place you imply they are durations, but that doesn't make sense to me...
